packages/module/package.json

```diff
@@ -13,8 +13,8 @@
     "clean": "rimraf dist",
     "docs:develop": "pf-docs-framework start",
     "docs:build": "pf-docs-framework build all --output public",
-    "docs:serve": "pf-docs-framework serve public --port 5000",
-    "docs:screenshots": "pf-docs-framework screenshots --urlPrefix http://localhost:5000",
+    "docs:serve": "pf-docs-framework serve public --port 5001",
```

Comment:
Port 5000 can be in use by MacOS.  See https://nono.ma/port-5000-used-by-control-center-in-macos-ventura-controlce